freeipa.spec.in

```diff
@@ -870,16 +874,42 @@ find \
 # TODO: workaround: some scripts are copied over, so the are always py2.
 # We have to explicitly set python3 here for ported files here
 PY3_SUBST_PATHS='
+client/ipa-certupdate
+client/ipa-client-automount
+client/ipa-client-install
+install/certmonger/ipa-server-guard
+install/certmonger/dogtag-ipa-ca-renew-agent-submit
+install/oddjob/com.redhat.idm.trust-fetch-domains
+install/tools/ipa-advise
+install/tools/ipa-adtrust-install
 install/tools/ipa-backup
+install/tools/ipa-ca-install
+install/tools/ipa-cacert-manage
 install/tools/ipa-compat-manage
+install/tools/ipa-csreplica-manage
+install/tools/ipa-custodia
 install/tools/ipa-dns-install
+install/tools/ipa-httpd-kdcproxy
+install/tools/ipa-kra-install
+install/tools/ipa-ldap-updater
 install/tools/ipa-managed-entries
 install/tools/ipa-nis-manage
+install/tools/ipa-otptoken-import
+install/tools/ipa-pkinit-manage
+install/tools/ipa-pki-retrieve-key
+install/tools/ipa-replica-conncheck
+install/tools/ipa-replica-install
+install/tools/ipa-replica-manage
 install/tools/ipa-replica-prepare
+install/tools/ipa-restore
+install/tools/ipa-server-certinstall
+install/tools/ipa-server-install
+install/tools/ipa-server-upgrade
+install/tools/ipa-winsync-migrate
 install/tools/ipactl
 '
 for P in $PY3_SUBST_PATHS; do
-    sed -i -e '1 s|^#!.*\bpython[^ ]*|#!%{__python3}|' $P
+    sed -i -e '1 s|^#!.*\bpython[0-9]*\(\s\+-\)\?|#!%{__python3} -bb|' $P
```

Comment:
It is. The commit states that this should be removed before pushing to production by which I mean a stable release.